server/schema/comment/comment.model.js

```diff
@@ -9,7 +9,7 @@
  * @since 0.1.0
  */
 
-const { Schema, model } = require('mongoose');
+const { Schema, model, Model: _Model } = require('mongoose');
```

Comment:
@Shurtu-gal why this change was done?

Reply:
This is to get the required intellisense using JSDoc during work.